### PR TITLE
added command deprecated message

### PR DESCRIPTION
**Pull request by Valansch**

### CODE REVIEW

File: features/chat_triggers.lua [hunk 1]
```diff
@@ -131,6 +131,15 @@ local function mentions(event)
     end
 end
 
+--warns admins that /c /command and /silent-command are deprecated
+local function notify_deprecated_command(event)
+    local command = event.command
+    if command == 'silent-command' or command == 'c' or command == 'command' then
```
Comment (linaori) [inline features/chat_triggers.lua:137]: What about a table to make this more generic?
```lua
local deprecated_commands = {
    ['silent-command'] = 'sc',
    ['c'] = 'sc',
    ['command'] = 'sc',
}
```

**Review by linaori — APPROVED** [review]

**Comment by grilledham** [comment]
Why does `/sc` deprecate `/c` or `/command` they serve different purposes?

**Comment by Valansch** [comment]
sc's only purpose is to deprecate /c and /command.
We dont want errors go to all players.

**Comment by grilledham** [comment]
I understand `/sc` replacing`/silent-command` but I think it follows that it replaces `/c` or `/command`. If you use `/c ` it's because you want people to see that you are using the command, so wouldn't you want people to also see the errors?

For example you might use `/c game.speed = 0.8` so players know you have changed the game speed. You wouldn't want to then receive a message saying it's deprecated.

**Comment by Valansch** [comment]
Yea you are right. I rest my case :)

**Comment by Valansch** [comment]
Thinking about it... I remember why i wanted to deprecate /c too.
I cannot disable /silent-command without disabling /c and /command too. And that was my plan...

**Pull request closed without merging**